content/learn/book/welcome/why-bevy/_index.md

```diff
@@ -5,8 +5,71 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: discuss the cool features and design philosophy
+There are plently of great reasons to use Bevy, but here are just a few:
+
+### ECS 
+**TL;DR: Better workflow and abstraction in your code.**
+
+**LR:**
+
+Bevy has been designed to use a flexible data system in order to provide a smart, usable workflow. The engine uses an **Entity Component System** (ECS) to acheive this goal. In traditional game engines, a singular object (entity) contains its logic and components in one system. This is known as an object-oriented entity system.
+
+This has its benifits, however, in general, it is being phased out by ECSs. Instead of containing all the systems and components in a singular entity, in ECS, all of this is abstracted into structs (in Rust) and functions. Entities are defined as unique integers. 
+
+Components are structs that contain values for said components. Systems are Rust functions that are parsed into the main function and can specify components that an entity must have to be affected by the system.
+
+**This can allow for a great amount of abstraction that help with S.O.L.I.D principles.**
+
+### Modularity
+
+**TL;DR: You can add and/or remove components you don't use.**
+
+**LR:**
+
+Bevy hinges on its plugin system, which allows for the most minimal of programs to be actually minimal in size. For example, if you are developing a console-based game, you may not need to include the ```WindowPlugin```, or if your game does not include audio, you can exclude the ```AudioPlugin```.
+
+In addition, there are more plugins made by our excellent community, and you could even make your own with ease, if the one you are looking for doesn't exist.
+
+This can potentially reduce the size of your application or game and give the engine even more functionality than built-in.
+
+### Hot Reloading
+
+**TL;DR: Easy file changes without recompilation**
+
+**LR:** 
+
+The Bevy Game Engine uses a **scene system** that allows for **hot reloading**. In very short, not technological terms, you have the ability to instantly change certain files (meshes, textures, scenes) and view the changes without recompiling.
+
+This allows for quick iteration for artists and 3D modelers as well as less compile times.
+
+### Much more!
+
+There is much more Bevy is unique for, including,
+- Fast Compile Times
+- Modular, parallel render graph
+- Flex Box Inspired UI system
```

Comment:
I'm not sure this feature is ready enough to be advertised (nor is flex-box UI very unique, even among game engines).